ha/util/message_bus.py

```diff
@@ -144,6 +146,16 @@ def stop(self):
 
 class MessageBus:
     ADMIN_ID = "ha_admin"
+    _DELIM = ">"
+    HA_GLOBAL_INDEX = "ha_conf"
+
+    @staticmethod
+    def init():
+        """
+        Initialize utils MessageBus Library with kafka endpoints
+        """
+        message_server_endpoints = Conf.get(MessageBus.HA_GLOBAL_INDEX, f"kafka_config{MessageBus._DELIM}endpoints")
+        utils_message_bus.init(message_server_endpoints)
```

Comment:
This(MessageBus.init) should be called only once. And not for every client.

Reply:
As in our message_bus wrapper all methods are static which are called directly on class name. because of that I am calling init() in every message_bus method.

Reply:
I hope the utlis_message_bus.init is idempotent, Selva can you please confirm. If so then we are ok to call init multiple times. Other part here is initializing message_server_endpoints, which I guess is ok if that happens multiple times. We will postpone refactoring this code to non-static class method member in future if needed, so has to do multiple init calls as of now.

Reply:
Sorry for confusion, I saw it is now called once per service. Resolving the comment.

Reply:
In current Implementation MessageBus.init() called once per service.
As per last chat/discussion resolving this comment. Thanks!